getting_started_scenarios/ecr_scenario/README.md

```diff
@@ -0,0 +1,60 @@
+# Amazon Elastic Container Registry Service Basic Scenario
+
+## Overview
+
+ This Amazon Elastic Container Registry Service (Amazon ECR) basic scenario demonstrates how to interact with the Amazon ECR service using an AWS SDK.  The scenario covers various operations such as creating an Amazon ECR repository, pushing images to the repository, setting an ECR repository policy, and so on. Here are the 6 top service operations this scenario will cover. 
+
+1. **Create an ECR repository**: The program prompts the user to enter a name for the ECR repository, then creates the repository using the `createECRRepository` method.
```

Comment:
The naming and casing on these actions looks Java specific. For example:
```suggestion
1. **Create an ECR repository**: The program prompts the user to enter a name for the ECR repository, then creates the repository using the `CreateRepository` action.
```
createECRRepository is Java specific. In JS it's not even a method, it's a command that's passed to a method: [CreateRepositoryCommand](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/ecr/command/CreateRepositoryCommand)